HAL/classes/fit/thomasfermi2D.py

```diff
@@ -10,27 +10,30 @@
 
 # -- global
 import numpy as np
+from collections import namedtuple
 
 # -- local
 from HAL.classes.fit.abstract import Abstract2DBellShaped
 
 
 # %% FUNCTIONS
-def TFParab(x, y, size_x, size_y, center_x, center_y):
-    return (
-        1
-        - (x - center_x) ** 2 / size_x ** 2
-        - (y - center_y) ** 2 / size_y ** 2
+def ThomasFermi2D(parameters):
+    """
+    Returns the Thomas-Fermi distributrion in 2D.
```

Comment:
"distribution" :wink: 
```suggestion
    Returns the integrated Thomas-Fermi distribution in 2D.
```